cluster/gce/gci/configure-helper.sh

```diff
@@ -980,6 +1010,7 @@ function start-kube-controller-manager {
   if [[ -n "${CA_KEY:-}" ]]; then
     params+=" --cluster-signing-cert-file=/etc/srv/kubernetes/ca.crt"
     params+=" --cluster-signing-key-file=/etc/srv/kubernetes/ca.key"
+    params+=" --insecure-experimental-approve-all-kubelet-csrs-for-group=node-bootstrapper"
```

Comment:
> Doesn't that mean that anyone who can touch the CSR API can issue a cert for any node?

No. The API forces those fields to be set to the user that created the CSR object.

https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/certificates/certificates/strategy.go#L57-L81